schutzbot/update-schutzfile-images

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+import json
+import os
+import sys
+import urllib.request
+
+SCHUTZFILE = "Schutzfile"
+
+
+def images_main_commit_id():
+    token = os.environ.get("GITHUB_TOKEN")
+    req = urllib.request.Request("https://api.github.com/repos/osbuild/images/commits/main")
+    req.add_header("Accept", "application/vnd.github+json")
+    if token:
+        # this API request doesn't necessarily require a token, but let's use it if we have one
+        req.add_header("Authorization", f"Bearer {token}")
+    try:
+        with urllib.request.urlopen(req, timeout=30) as resp:
+            body = resp.read()
+    except urllib.error.HTTPError as http_error:
+        print(http_error)
```

Comment:
There's no real need to catch-and-print-and-exit. This is the same as default behavior so let's *not* catch the exception?